frontend/src/pages/LoginView.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Card } from '@components/ant-design/Card';
+import { Button } from '@components/ant-design/Button';
+import { Box } from '@components/atoms/Box';
+import { Paragraph, Title } from '@components/ant-design/Typography';
+import { LoginGraphic, Google } from '@components/svg';
+
+export const LoginView = () => {
+  const click = () => {
+    console.log('Button was clicked');
+  };
+
+  return (
+    <>
+      <Box style={{ textAlign: 'center', fontStyle: 'normal' }}>
+        <Card
+          pt={36}
+          pb={36}
+          width={606}
+          mt={88}
+          mb={49}
+          style={{ boxSizing: 'border-box', textAlign: 'center', marginLeft: 'auto', marginRight: 'auto' }}
+        >
+          <Title level={1} style={{ fontFamily: 'Roboto', paddingBottom: '36px', lineHeight: '36px' }}>
+            Witaj w aplikacji CodersBoard
+          </Title>
+          <Paragraph large style={{ fontFamily: 'Roboto', height: '22px', lineHeight: '24px' }}>
+            Zaloguj się za pomocą e-maila w domenie CodersCrew
+          </Paragraph>
+          <Button onClick={click} mt={24} style={{ height: '42px', borderRadius: '4px' }}>
```

Comment:
Ideą, dla której mamy wszystkie komponenty, jak button w folderze `components` jest to, żeby później nie musieć nadpisywać ich styli (w większości przypadków). W tym przypadku skorzystaj po prostu z Buttona z propsem `size="large"`